.buildkite/pipeline.yml

```diff
@@ -71,6 +71,34 @@ steps:
         notify:
           - github_commit_status:
               context: "Reader Unit Tests"
+      - label: "🔬 Keystone Unit Tests"
+        command: |
+          .buildkite/commands/shared-set-up.sh
+          xcodebuild \
+            -scheme Keystone \
+            -destination 'platform=iOS Simulator,OS=18.2,name=iPhone 16' \
+            test \
+            | xcbeautify
```

Comment:
Switching to fastlane to be consistent with how other tests execute?

Reply:
Good observation. To do this, we'd need a custom lane. Alternatively, we could do `bundle exec fastlane run scan...` but at that point, that's not that different from calling `xcodebuild`.

I didn't document that, but I see these steps as temporary only, just to get fast CI feedback for these foundational modules that are still in flux.